devsetup/edpm/edpm-play.yaml

```diff
@@ -172,10 +204,10 @@ spec:
                   edpm_nova_compute_service_user_username: nova
                   edpm_nova_compute_vnc_novncproxy_base_url: http://192.168.24.3:6080
                   edpm_ovn_dbs:
-                  - 192.168.24.1
+                  - _EDPM_OVN_DBS_
             vars:
                 registry_name: "quay.io"
-                registry_namespace: "tripleomastercentos9"
+                registry_namespace: "tripleozedcentos9"
```

Comment:
I think it should be based on Wallaby as tripleo master/zed containers are going to become unsupported shortly? @rlandy WDYT?

Reply:
our current work is based on zed. and we default to those also in the operators

Reply:
I confused this to the Wallaby base used for adoption, sorry